showcase/src/assets/examples/collapsible/collapsible-custom1.html

```diff
@@ -0,0 +1,31 @@
+<stark-collapsible
+		htmlId="id0"
```

Comment:
Let's provide more meaningful IDs to our variables/components to prevent developers from developing those bad practices.